.pre-commit-config.yaml

```diff
@@ -14,6 +14,8 @@ exclude: |
   ^docs/_templates/.*\.html$|
   # Don't bother non-technical authors with formatting issues in docs
   readme/.*\.(rst|md)$|
+  # Ignore test files in addons
+  /tests/fixtures/*|
```

Comment:
final consensus for this one is "samples"
https://github.com/OCA/oca-addons-repo-template/pull/240#discussion_r1429685636